### ISSUE TITLE
Is it possible to export psd data as a tree structure?

### ISSUE THREAD

**Issue by MonkeyOliver** [psd-tools/psd-tools#257]
Just like `psd.tree()` in [psd.js](https://github.com/meltingice/psd.js) or `psd.tree` in [psd.rb](https://github.com/layervault/psd.rb).

I couldn't find such a feature in the document.

**Comment by kyamagu** [comment]
No, there is no such API. The low-level PSD structure is fairly complicated and cannot be easily represented by a simple tree.
